src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -859,6 +875,16 @@ private TerminalKeyInfo(TermInfo.Database db)
                     AddKey(db, TermInfo.Database.KeySRight, ConsoleKey.RightArrow, true, false, false);
                     AddKey(db, TermInfo.Database.KeyUp, ConsoleKey.UpArrow);
 
+                    AddPrefixKey(db, "kLFT", ConsoleKey.LeftArrow);
```

Comment:
Perhaps they could go down near the other constants below? although those seem to be for well-known key indices and these are a little different.